src/ray/raylet/node_manager.cc

```diff
@@ -13,6 +14,20 @@ namespace {
 #define RAY_CHECK_ENUM(x, y) \
   static_assert(static_cast<int>(x) == static_cast<int>(y), "protocol mismatch")
 
+/// Macro to handle early return for preprocessing.
+/// An early return will take place if the worker is being killed due to the exiting of
+/// driver, or the worker is not registered yet.
+#define PREPROCESS_REQUEST(REQUEST_TYPE, REQUEST, SEND_REPLY)                       \
+  WorkerID worker_id = WorkerID::FromBinary(REQUEST.worker_id());                   \
+  do {                                                                              \
+    if (!PreprocessRequest(worker_id, #REQUEST_TYPE)) {                             \
```

Comment:
Can we name this and the macro something more descriptive of what the macro does? Something like `RETURN_IF_WORKER_DEAD` and `CheckWorkerDead`?

Reply:
This macro does some common preprocessing work for every worker requests. Currently it does 2 things: logging request and checking worker dead. And we may need it to do more things in the future. So `CheckWorkerDead` doesn't describe the whole purposes. 
I think we can name this `PREPROCESS_WORKER_REQUEST` to be more specific.